api/static/index.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>openyieldtables API</title>
+    <style>
+      body {
+        font-family: "Courier New", Courier, monospace;
+        margin: 10vh 10vw;
+      }
+    </style>
+  </head>
+  <body>
+    <h1>Welcome to the openyieldtables API</h1>
+    <br />
+    <h4>This is the API for the openyieldtables project.</h4>
+    <p>
+      The documentation of the project can be found
+      <a href="https://openyieldtables.readthedocs.org/en/latest">here</a>. The
+      API documentation can be found
+      <a href="https://yieldtables.org/docs">here</a>.
```

Comment:
You could link to this relatively since it runs on the same host:
```suggestion
      <a href="/docs">here</a>.
```